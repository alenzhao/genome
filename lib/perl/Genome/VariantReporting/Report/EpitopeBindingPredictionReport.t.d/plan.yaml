reports:
    'epitope-binding-prediction':
        interpreters:
            'epitope-variant-sequence': {}
