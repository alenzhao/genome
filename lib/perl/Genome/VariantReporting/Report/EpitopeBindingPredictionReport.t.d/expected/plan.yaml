---
experts: {}
needs_translation: 1
reports:
  epitope-binding-prediction:
    filters: {}
    interpreters:
      epitope-variant-sequence: {}
